shared/src/business/useCases/judgeActivityReport/getOpinionsFiledByJudgeInteractor.ts

```diff
@@ -87,5 +87,15 @@ export const getOpinionsFiledByJudgeInteractor = async (
 
   const sortedResult = orderBy(result, 'eventCode', 'asc');
 
-  return sortedResult;
+  await applicationContext.getNotificationGateway().sendNotificationToUser({
+    applicationContext,
+    clientConnectionId: params.clientConnectionId,
+    message: {
+      action: 'fetch_opinions_complete',
+      opinions: sortedResult,
+    },
+    userId: authorizedUser.userId,
+  });
+
+  return sortedResult || [];
```

Comment:
Since this is an async endpoint, the result will be returned via websocket message above. I may be incorrect but I believe that means the return value is not consumed and can be removed.

Reply:
@rachelschneiderman 

You are correct. The reason I added them was because integration tests failures. The items return from the call weren't been set in state correctly "in time". I have made the changes to remove the return statements and added `waitForLoadingComponentToHide` to wait for the calls to be resolved.